### PR TITLE
Sync various sections

### PR DESCRIPTION
**Pull request by saundefined**

### CODE REVIEW

File: reference/pdo/pdo/prepare.xml [hunk 1]
```diff
@@ -75,6 +74,11 @@
     supporté nativement par le pilote.
    </simpara>
   </note>
+  <para>
+   Depuis PHP 7.4.0, les points d'interrogation peuvent être échappés en les doublant.
```
Comment (Girgias) [inline reference/pdo/pdo/prepare.xml:78]:
```suggestion
   À partir de PHP 7.4.0, les points d'interrogation peuvent être échappés en les dédoublant.
```

File: reference/pdo/pdo/prepare.xml [hunk 2]
```diff
@@ -160,6 +165,21 @@ $red = $sth->fetchAll();
 $sth->execute(array(175, 'jaune'));
 $yellow = $sth->fetchAll();
 ?>
+]]>
+    </programlisting>
+   </example>
+   <example>
+    <title>Modèle de déclaration SQL avec point d'interrogation échappé</title>
```
Comment (Girgias) [inline reference/pdo/pdo/prepare.xml:172]:
```suggestion
    <title>Modèle de déclaration SQL avec un point d'interrogation échappé</title>
```

**Review by Girgias — COMMENTED** [review]
Thank you! Other than a couple nits and implementing the changes where ``never`` is now mentionned. LGTM

File: appendices/reserved.xml [hunk 3]
```diff
@@ -556,6 +556,9 @@
         <entry>
          object (à partir de PHP 7.2)
         </entry>
+        <entry>
+         mixed (à partir de PHP 8.0)
+        </entry>
```
[the remark this answers is not in the document]
Comment (saundefined) [reply appendices/reserved.xml:561]: Sure, updated

**Pull request merged**